extension/src/popup/components/hardwareConnect/LedgerSign/index.tsx

```diff
@@ -0,0 +1,144 @@
+import React, { useState, useEffect, useRef } from "react";
```

Comment:
we now have a LedgerSign component (that slides up when signing a transaction), and a LedgerConnect one (it's own view for when initially connecting)